packages/itinerary-body/src/defaults/route-description-footer.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/prefer-default-export */
+import React, { ReactElement } from "react";
+import * as S from "../styled";
+import { RouteDescriptionFooterProps } from "../types";
+
+export function DefaultRouteDescriptionFooter({
```

Comment:
Can we change the path on this file to be in the stories directory? Any non-localized code should be in the stories folder.

Reply:
Done.